src/consensus/pbft_manager.cpp

```diff
@@ -1691,8 +1574,8 @@ bool PbftManager::giveUpNextVotedBlock_() {
 void PbftManager::countVotes_() {
```

Comment:
what do we use this function for ?

Reply:
We use this function to debug number of votes for each step